src/loader.ts

```diff
@@ -952,7 +1007,14 @@ interface ModuleDefinitionArguments extends Array<any> {
 				}
 				else {
 					let parentMid = (parent ? ' (parent: ' + parent.mid + ')' : '');
-					throw new Error('Failed to load module ' + module.mid + ' from ' + url + parentMid);
+					let message = `Failed to load module ${module.mid} from ${url}${parentMid}`;
```

Comment:
This block of code repeats the code at line 957. Could this be abstracted so it's not repeated?